src/parser/jobs/rdm/Gauge.js

```diff
@@ -184,6 +192,50 @@ export default class Gauge extends Module {
 		}
 
 		_onComplete() {
+			if (this._missingAreo || this._missingThunder) {
+				this.suggestions.add(new Suggestion({
+					content: <Fragment>
+						<Message warning icon>
+							<Icon name="warning sign"/>
+							<Message.Content>
+									Due to a missing cast at the start of the log, mana calculations might be off.
+									Additionally 1 or more finishers might have been incorrectly flagged as wrongly used.
+							</Message.Content>
+						</Message>
+					</Fragment>,
+					severity: SEVERITY.MAJOR,
+					why: <Fragment>
+						You were the first damage event, so it doesn&apos;t log your first cast as cast by you
+					</Fragment>,
+				}))
+			}
+
+			if (this._missingAreo) {
```

Comment:
Having 3 suggestions for it seems like overkill. Probably should be condensed into 1.